delivery/models.py

```diff
@@ -8,3 +8,13 @@ class Restaurant:
     work_time: str = ""
     phone_number: str = ""
     id: str = ""
+
+
+@dataclass
+class User:
+    login: str = ""
+    password: str = ""
+    name: str = ""
+    phone_number: str = ""
+    rights: str = ""
```

Comment:
`rights` should be an array, I guess

Reply:
I think that rights is like 'user', 'admin'. And one person have only one variant because admin right include all user rights and extra

Reply:
then it sounds like a `role`

Reply:
What will be better? Change type or leave type and change name?